modules/scale-down-data-plane.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="scale-down-data-plane_{context}"]
+= Scaling down the data plane to zero
+
+If you are not using the hosted control plane, to save the resources and cost you can scale down a data plane to zero.
+
+.Procedure
+
+. Set the `kubeconfig` file to access the hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ export KUBECONFIG=<install_directory>/auth/kubeconfig
+----
+
+. Get the name of `NodePool` associated to your hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get nodepool --namespace <HOSTED_CLUSTER_NAMESPACE>
+----
+
+. Optional: To prevent the pods from draining, add the `nodeDrainTimeout` field in the `NodePool` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit NodePool <nodepool> -o yaml --namespace <HOSTED_CLUSTER_NAMESPACE>
+----
++
+.Example output
+[source,terminal]
+----
+spec:
+  arch: amd64
+  clusterName: clustername <1>
+  management:
+    autoRepair: false
+    replace:
+      rollingUpdate:
+        maxSurge: 1
+        maxUnavailable: 0
+      strategy: RollingUpdate
+    upgradeType: Replace
+  nodeDrainTimeout: 0s <2>
+----
+<1> Defines the name of your hosted cluster.
+<2> Specifies the total amount of time that the controller spends to drain a node. The default value is `0` seconds. By default, the `nodeDrainTimeout: 0s` setting blocks the node draining process.
++
+[NOTE]
+====
+If you want to allow the node draining process for a certain period of time, you can set the value of the `nodeDrainTimeout` field accordingly, for example, `nodeDrainTimeout: 1m`.
```

Comment:
Just want to highlight that "want" is a use with caution word in SSG https://redhat-documentation.github.io/supplementary-style-guide/#want 

```suggestion
You can set the value of the `nodeDrainTimeout` field accordingly, for example, `nodeDrainTimeout: 1m` to allow the node draining process to continue for a certain period of time.
```